indica/operators/spline_fit.py

```diff
@@ -107,6 +106,8 @@ def example_run(
     knots: list = None,
     plot: bool = True,
 ):
+    from indica.readers.read_st40 import ReadST40
```

Comment:
For info, I'm moving all the examples to a new /examples/ directory, so we can decouple the drivers from the examples. But this can stay here for the time being.

Reply:
Makes sense, I only moved the import inside the function as `ReadST40` failed to import if (like me) you don't have MDSPlus installed, so this just scoped the import to not block importing from `spline_fit` in general